asset/filtered_manager.go

```diff
@@ -30,8 +30,14 @@ type filteredManager struct {
 func (f *filteredManager) Get(ctx context.Context, asset *corev2.Asset) (*RuntimeAsset, error) {
 	var filteredAsset *corev2.Asset
 
+	fields := logrus.Fields{
+		"entity":  f.entity.Name,
+		"asset":   asset.Name,
+		"filters": asset.Filters,
+	}
+
 	if len(asset.Builds) > 0 {
-		fields := logrus.Fields{
+		fields = logrus.Fields{
 			"entity": f.entity.Name,
 			"asset":  asset.Name,
 		}
```

Comment:
It appears we no longer need this assignation, since these fields are already defined on lines 33-37.

Reply:
I didn't want to display the top-level asset's filters as they don't do anything in the case where an asset has builds.